wagtailio/project_styleguide/templates/patterns/components/streamfields/quotes.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailcore_tags %}
+
+<div class="quotes">
+    <div class="quotes__header">
+        <p class="quotes__title heading-one">{{ value.title }}</p>
+        <div>
+            <svg aria-hidden="true" class="quotes__icon"><use xlink:href="#quotes"></use></svg>
+        </div>
+    </div>
+    <div class="quotes__grid">
+        {% for item in value.items.all %}
+        <div>
+            <p class="heading-four">{{ item.quote }}</p>
```

Comment:
question: is there a reason not use [`<blockquote>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)? It would most definitely be more semantic

Reply:
I can't see one, I've updated this.